### ISSUE TITLE
[Controller] Fight action button - stand ground by default

### ISSUE THREAD

**Issue by blaze-form3** [diasurgical/devilutionX#6436]
### Operating System

Windows x64

### DevilutionX version

1.5.0 (latest release)

### Describe

First of all great job! I wouldn't believe that I will have so much fun playing Diablo 1 again in 2023 using my Xbox Controller.

It feels like the controller action button is a simple mapping from mouse left click. When fighting with a melee weapon the character is running around to closest monsters with pressing only the action button; and there is no need to control the movement.
It is understandable when using a mouse, cause you need to click on a target monster but with a controller it feels too easy and bit off.
I know that there is stand ground button but of course I am not pushing it and exploiting the easy mode.

My suggestion is to change this behaviour to make my character move only when I press a movement button;
or have stand ground by default unless I press a movement button.

### To Reproduce

1. Take a melee weapon
2. Enter any dungeon
3. Find a group of monsters
4. Press action button continuously 
5. The char is running from monster to monster and there is no need to control the movement
6. Feels too easy and off - no other game do that

### Expected Behavior

The character should move only when I use a movement button

**Comment by StephenCWills** [comment]
I think this request is pretty similar to what's already been planned for 1.6.0. We need to avoid moving the character without explicit intent for visually impaired players so they don't get lost as a result.

> Without auto-targeting, I could see this being extremely painful to play unless there was a new mechanic to "turn" the hero without moving him (which would be interesting, I guess).

This already exists. It's the Stand Ground function.

> ... or have stand ground by default unless I press a movement button.

There is a Toggle Stand Ground feature which is basically what you're asking for here. Unless you mean you want to have to hold a button in order to move your character?

**Comment by blaze-form3** [comment]
Maybe I didn't make myself clear. There are two things that feels off while using the controller.

1. **Auto-targeting** that @julealgon refers to in his comment. I agree this is OP and there should be some kind restriction to target only the enemies the character is facing. But this is not what this issue is about and I can imagine this could be tricky to implement.
2. Melee **"auto-chasing"** - this is what this issue is about. My character is auto-chasing auto-targeted enemies which feels like an easy mode.

I am aware of Stand Ground and Toggle Stand Ground but it's not the point. The stand ground makes sense when you're using a mouse and left click is bound to both move and attack, but it is not a case when using a controller. Controller has stick dedicated only to movement and separate attack button so stand ground shouldn't be the case. Stick should move and attack should attack.

To sum up, when using a controller:
- attack action shouldn't move your character toward targeted enemy
- movement should only be triggered by the LS
- stand ground and toggle stand ground shouldn't even exist for controller schema

**Comment by julealgon** [comment]
> Melee "auto-chasing" - this is what this issue is about. My character is auto-chasing auto-targeted enemies which feels like an easy mode.

I don't understand why this feels like "easy mode" considering you can do the exact same thing with a mouse and keyboard though.

Not saying I necessarily disagree with your general proposal, but difficulty is not an argument I'd use to justify it personally.
